src/utils/api-request.ts

```diff
@@ -224,11 +224,16 @@ export class SignedApiRequestHandler extends HttpRequestHandler {
       port: number,
       path: string,
       httpMethod: HttpMethod,
-      data: Object,
-      headers: Object,
-      timeout: number): Promise<Object> {
+      data?: Object,
+      headers?: Object,
+      timeout?: number): Promise<Object> {
     return this.app_.INTERNAL.getToken().then((accessTokenObj) => {
-      let headersCopy: Object = deepCopy(headers);
+      let headersCopy: Object;
```

Comment:
Suggestion: You can also do:
`let headersCopy: Object = (headers && deepCopy(headers)) || {};`

Reply:
Much cleaner. Thanks.